connectors/src/connectors/google_drive/temporal/file.ts

```diff
@@ -472,8 +472,8 @@ async function upsertGdriveDocument(
   if (documentLen > 0 && documentLen <= maxDocumentLen) {
     const parents = (
       await getFileParentsMemoized(connectorId, oauth2client, file, startSyncTs)
-    ).map((f) => f.id);
-    parents.push(file.id);
+    ).map((f) => getDocumentId(f.id));
```

Comment:
getDocument / documentId are the prefixed version right? 

how did it work in the past?

Not sure I understand how it worked in the first place?

Reply:
> getDocument / documentId are the prefixed version right?

Yes

Reply:
So we used to upsert without the `gdrive` prefix an now we want to upsert with it? I've not been following along but we want to align the id between core and connectors, is this correct?